.github/workflows/translation.yml

```diff
@@ -0,0 +1,81 @@
+name: "Refresh translations"
+
+on:
+  workflow_call:
+    secrets:
+      tx_token:
+        required: true
+      based_branch:
+        required: true
+
+jobs:
+  ci:
+    name: "Translations"
+    runs-on: ubuntu-latest
+    steps:
+
+    ##########################################
+    # Checkout repository                    #
+    ##########################################
+    - name: "Checkout"
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ secrets.based_branch }}
+        fetch-depth: 0 #To Fetch All Tags and Branches
+
+    ##########################################
+    # Install depdendencies                  #
+    ##########################################
+    - name: Install Dependencies
+      uses: php-actions/composer@v6
+      with:
+        dev: yes
+
+    - name: Install xgettext
+      run: sudo apt-get install gettext libgettextpo-dev
+
+    ##########################################
+    # Generate <plugin>.pot file             #
+    ##########################################
+    - name: "Generate <plugin>.pot file"
+      run: ./vendor/bin/extract-locales
+
+
+    ##########################################
+    # Push and Pull transifiex source        #
+    ##########################################
+    - name: Push source file (.pot)
+      uses: transifex/cli-action@v2
+      with:
+        args: push
+        token: ${{ secrets.tx_token }}
+
+    - name: Clean Transifex CLI
+      run: rm -fr /tmp/tx
+
+    - name: Pull source file (.po / .mo)
+      uses: transifex/cli-action@v2
+      with:
+        args: pull --force --all --minimum-perc=80
+        token: ${{ secrets.tx_token }}
+
+
+    ##########################################
+    # Create Pull Request                    #
+    ##########################################
+    - name: Create Pull Request
+      id: cpr
+      uses: peter-evans/create-pull-request@v7
+      with:
+        commit-message: Update report
+        committer: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
+        author: ${{ github.actor }} <${{ github.actor_id }}+${{ github.actor }}@users.noreply.github.com>
+        signoff: false
+        branch: transifex/i18n-updates
```

Comment:
The branch name should maybe have a randomized name. Indeed, if a previous PR is still not merged, it will probably result in a branch name conflict.

Reply:
done !